src/components/dialog/faultEvent/FaultEventCreation.tsx

```diff
@@ -181,18 +189,20 @@ const FaultEventCreation = ({ useFormMethods, isRootEvent, eventValue, isEditedE
               />
 
               {/* Probability field */}
-              <TextField
-                label={t("newFtaModal.probability")}
-                type="number"
-                min={0}
-                max={1}
-                step={0.01}
-                error={!!errors.probability}
-                helperText={errors.probability?.message}
-                className={classes.probability}
-                defaultValue=""
-                {...register("probability")}
-              />
+              {!isEditMode && eventTypeWatch === EventType.BASIC && (
```

Comment:
I do not understand why this is not related to EventType.EXTERNAL but it is related to EventType.BASIC